en/setup/dev_env_mac.md

```diff
@@ -45,7 +45,7 @@ Install pip if you don't already have it and use it to install the required pack
 
 ```sh
 sudo easy_install pip
-sudo -H pip install pyserial empy toml numpy pandas jinja2 pyyaml
+sudo -H pip install nose tornado pyserial empy toml pandas jinja2 pyyaml
```

Comment:
Hi @unipheas ,
I know nothing about mac. We should use whatever libraries are used in the docker mac environments. Are nose and tornado what they use?

Reply:
After using a clean MacBook (I wiped mine) I had to install `nose` and `tornado` as they where required by other packages to run. I'm not sure about what's in the docker Mac environment. With the most recent updates, this could have been taken care of.

I'm planning on doing another wipe the beginning of next month. I can go through this process again to make sure everything works from scratch. I may need to get myself another MacBook for things like this 😛

Reply:
Thanks @unipheas . I thought that dependencies like that were automatically pulled in? Anyway I am loath to make this change unless it matches CI.

@julianoes Do you use macOS?

Reply:
> @julianoes Do you use macOS?

Not unless I have to.

I never installed these two. What were they required for?

Reply:
When I first setup these were required/missing packages by other packages that are being installed. I think it has something to do with empty or toml. I can't remember without going through the entire process again. 

I'm trying to make it easier for the user to setup instead of having to debug all the errors because they haven't ever setup a development environment or have something else setup.

Reply:
That is very reasonable, but they should not be needed; ie packages for numpy, toml are supposed to pick up their own dependencies. We need someone with a couple of recent clean macOS to test this.

Reply:
In a few weeks I will completely wipe my MacBook and can try again.

Reply:
Really appreciated. It is possible that what worked in previous versions might have changed, but either way we'd need to establish where the problems lie.

Reply:
No problem. Jinger and I are doing a workshop today so we'll see what kind of issues people have on fresh installs. That said, I'm having one of the organizers go through the process and they are running into problems already because of Python. Once I get more information then I'll update here with what's going on.